### PR TITLE
Add FlagoftheCharles plugin.

### PR DESCRIPTION
**Pull request by irstacks**
Display's Boston's Community Boating, Inc. current weather flag as well as some related weather information.

### CODE REVIEW

File: Weather/FlagoftheCharles.15m.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/ruby
+
+# <bitbar.title>Flag of the Charles</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Isaac</bitbar.author>
+# <bitbar.author.github>irstacks</bitbar.author.github>
+# <bitbar.desc>Shows Boston's current Community Boating, Inc flag and related weather information.</bitbar.desc>
+# <bitbar.image>http://www.community-boating.org/wp-content/themes/communityboating/images/burgee.png</bitbar.image>
```
Comment (keithamus) [inline Weather/FlagoftheCharles.15m.rb]: This should be a screenshot of the plugin working. Could you amend this please 😄

File: Weather/FlagoftheCharles.15m.rb [hunk 2]
```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/ruby
+
+# <bitbar.title>Flag of the Charles</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Isaac</bitbar.author>
+# <bitbar.author.github>irstacks</bitbar.author.github>
+# <bitbar.desc>Shows Boston's current Community Boating, Inc flag and related weather information.</bitbar.desc>
+# <bitbar.image>http://imgur.com/gQ3suo0</bitbar.image>
```
Comment (keithamus) [inline Weather/FlagoftheCharles.15m.rb]: Can you please change this to point to the actual PNG, so `https://i.imgur.com/gQ3suo0.png`

File: Weather/FlagoftheCharles.15m.rb [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+hobo_weather_html = Nokogiri.HTML(open(HOBO_WEATHER_PAGE))
+
+stats = {}
+# => {"Wind Speed:"=>"3.4", "Gust Speed:"=>"10.2", "Wind Direction:"=>"NNE 20", "Rain:"=>"0.00", "Air Temp:"=>"48.41", "Water Temp:"=>"52.86", "Pressure:"=>"29.880", "PAR:"=>"384", "RH:"=>"99.00", "Dew Point:"=>"48.15", "Battery:"=>"4.415"}
+
+units = {}
+# => {"Wind Speed:"=>"mph", "Gust Speed:"=>"mph", "Wind Direction:"=>"°", "Rain:"=>"in", "Air Temp:"=>"°F", "Water Temp:"=>"°F", "Pressure:"=>"inHg", "PAR:"=>"uE", "RH:"=>"%", "Dew Point:"=>"°F", "Battery:"=>"V"}
+
+hobo_weather_html.css('.latest-conditions-info').each do |con|
+	_label = con.css('span.latest-conditions-info-label')[0].text
```
Comment (keithamus) [inline Weather/FlagoftheCharles.15m.rb]: Rubocop (our syntax linter as part of our build) doesn't like underscore prefixes, could you please change this to `label` instead?

File: Weather/FlagoftheCharles.15m.rb [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+	_reading = con.css('span.latest-conditions-info-reading')[0].text
+	_units = con.css('span.latest-conditions-info-units')[0].text
```
Comment (keithamus) [inline Weather/FlagoftheCharles.15m.rb]: (These will need to change too)

File: Weather/FlagoftheCharles.15m.rb [hunk 5]
```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/ruby
```
Comment (keithamus) [inline Weather/FlagoftheCharles.15m.rb]: Can you make sure this file is executable - `chmod +x Weather/FlagoftheCharles.15m.rb; git add Weather/FlagoftheCharles.15m.rb; git commit -m'make FlagoftheCharles.15m.rb executable'` should do the trick 😄

Comment (irstacks) [inline Weather/FlagoftheCharles.15m.rb]: on it!

**Pull request merged**